### PR TITLE
69 refactor tile size

### LINKED ISSUE #69: Refactor tileSize
Refactor tileSize to be public static and change all the usages to use the static value rather than passing it around.

### PR DESCRIPTION
**Pull request by Matte02**
## Summary
Give Pawntastic public static getters for all public static variables and change those to private static.

## Test Coverage
imageimage

## Diagrams
Will be changed and added if the code gets approved.

Closes #69

### CODE REVIEW

File: src/main/java/com/thebois/views/game/ViewableResource.java [hunk 1]
```diff
@@ -17,21 +16,15 @@
     /**
      * Viewable Water tile.
      */
-    WATER(
-        ResourceType.WATER,
-        TextureUtils.createSquareTexture((int) Pawntastic.TILE_SIZE, Color.valueOf("#1E90FF"))),
+    WATER(ResourceType.WATER, TextureUtils.createSquareTexture(Color.valueOf("#1E90FF"))),
```
[the remark this answers is not in the document]
Comment (Matte02) [reply src/main/java/com/thebois/views/game/ViewableResource.java:19]: Gave Texture utils a load of overload methods, hopefully its fine now.

File: src/main/java/com/thebois/views/TextureUtils.java [hunk 2]
```diff
@@ -85,26 +109,48 @@ public static Texture createSquareTexture(final int sideLength, final Color colo
...
      * Creates a new instance of a triangle shaped texture with the side length.
      *
-     * @param sideLength The size of the triangle texture, in pixels.
      * @param color      The color used in the texture.
+     * @param sideLength The length of every side in the triangle.
      *
      * @return The triangle texture.
      */
-    public static Texture createTriangleTexture(final int sideLength, final Color color) {
+    public static Texture createTriangleTexture(final Color color, final int sideLength) {
```
Comment (jonathan-lindqvist) [inline src/main/java/com/thebois/views/TextureUtils.java:153]: oke

**Review by jonathan-lindqvist — APPROVED** [review]
Better codewise, but the UML-diagrams 😩

**Comment by Matte02** [comment]
> but the UML-diagrams 😩

Ya, can't say I look forward to do these UML diagrams

**Pull request merged**